### PR TITLE
Publish images to opentelemetry namespaces

### LINKED ISSUE #14: Publish images to opentelemetry namespaces
Currently, the release is pushing container images to quay.io/jpkroehling, and it should publish to the official namespaces at quay and Docker Hub.

### PR DESCRIPTION
**Pull request by jpkrohling**
Fixes #14

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: .goreleaser.yaml [hunk 1]
```diff
@@ -35,12 +35,16 @@ builds:
       - goos: windows
         goarch: arm64
 docker_manifests:
-  - name_template: quay.io/jpkroehling/otelcol:{{ .Tag }}
+  - name_template: quay.io/opentelemetry/otelcol:{{ .Tag }}
```
Comment (bogdandrutu) [inline .goreleaser.yaml:38]: Why this? Who owns this?

Comment (jpkrohling) [reply .goreleaser.yaml:38]: Legacy. We created this when we didn't know who owned DockerHub's "opentelemetry" namespace. Later on, "otel" was created there and we stopped publishing to Quay, IIRC.

Comment (jpkrohling) [reply .goreleaser.yaml:38]: And these people have access to the repository: 

image

I should probably remove @objectiser and @pavolloffay from it, though, as they are not maintainers of this project.

Comment (pavolloffay) [reply .goreleaser.yaml]: I would like to see OTEL starting publishing to `ghcr.io` than quay or docker hub.

**Review by bogdandrutu — COMMENTED** [review]
GitHub Actions need to change as well

**Comment by jpkrohling** [comment]
PR updated!

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**